pt/config/config.py

```diff
@@ -15,8 +15,8 @@
 
 SSL_PATH = os.environ.get('SSL_PATH', '/home/energy-trading-platform/pt/certificate/')
```

Comment:
`/pt/certificate/`  may be better for the default value which is same as your `docker-compose.yml`.

Reply:
Fix in 1e9e48d.